lib/rules/web/admin_console/4.4/search.xyaml

```diff
@@ -0,0 +1,44 @@
+search_by_resource:
```

Comment:
`search_by_resource_type` and `search_by_api_group` need to be updated using this new rule

Reply:
And all these rules can be moved to `filter.xyaml`